packages/video_player/video_player_android/lib/src/android_video_player.dart

```diff
@@ -143,6 +143,7 @@ class AndroidVideoPlayer extends VideoPlayerPlatform {
             buffered: values.map<DurationRange>(_toDurationRange).toList(),
             eventType: VideoEventType.bufferingUpdate,
           );
+        case 'behindLiveWindow':
```

Comment:
Why is this new event necessary? Is ExoPlayer really starting buffering without calling the listener we have registered?